### ISSUE TITLE
[Feature] Filter feed

### ISSUE THREAD

**Issue by ItsIgnacioPortal** [jonathanklee/Sapio#5]
It would be nice to have a way to filter the feed.  

- Only show installed apps.
- Only show `bareAOSP` apps
- Only show `user` apps

 I think the user would enable these filters by tapping on a funnel icon in the top right corner, and then ticking checkboxes

**Comment by jonathanklee** [comment]
@ItsIgnacioPortal Indeed, this would be cool. There is a lot of improvements I would like to do. I'll implement filters for sure.
